h/config.py

```diff
@@ -60,6 +62,10 @@
     EnvSetting('nsq.reader.addresses', 'NSQ_READER_ADDRS'),
     EnvSetting('nsq.writer.address', 'NSQ_WRITER_ADDR'),
     EnvSetting('origins', 'ALLOWED_ORIGINS'),
+    EnvSetting('rabbitmq.address', 'RABBITMQ_ADDRESS'),
+    EnvSetting('rabbitmq.user', 'RABBITMQ_USER'),
+    EnvSetting('rabbitmq.password', 'RABBITMQ_PASSWORD'),
+    EnvSetting('rabbitmq.vhost', 'RABBITMQ_VHOST'),
```

Comment:
Might also be worth considering not reusing `BROKER_URL` (and possibly renaming it to `CELERY_BROKER`) since Celery does work with other message brokers than RabbitMQ, while our new real-time module only works with an amqp compatible message broker.